qdax/core/containers/repertoire.py

```diff
@@ -225,6 +225,7 @@ def add(
         batch_of_genotypes: Genotype,
         batch_of_descriptors: Descriptor,
         batch_of_fitnesses: Fitness,
+        batch_of_extra_scores: ExtraScores,
```

Comment:
Should we make it Optional with a default value to None?

Reply:
I agree with this! Will make it more clear as already mentioned in the PR that the default MapElitesRepertoire is not using it.